app/models/book.rb

```diff
@@ -4,6 +4,17 @@ class Book < ApplicationRecord
   belongs_to :category, optional: true
 
   scope :top_score, ->{order rate_score: :desc}
+  scope :sort_by_likes_count_asc,
```

Comment:
nếu define trên cùng 1 line thì sẽ dùng `->`, trên nhiều line thì dùng `(lambda do |args| ... end)`
fix ở dưới tương tự em nhé
```suggestion
  scope :sort_by_likes_count_asc, (lambda do
    left_joins(:likes).group(:id).order "COUNT(likes.id) ASC"
  end)
```